### PR TITLE
Start in animation tab levelbuilder setting

### PR DESCRIPTION
**Pull request by caleybrock**

### CODE REVIEW

File: apps/src/gamelab/stateQueries.js [hunk 1]
```diff
@@ -46,3 +46,12 @@ function countAllowedModes(state) {
...
 }
+
+/**
+ * Decide whether we should start in the Animation Tab on page load.
+ * @param {Object} state - Game Lab redux state.
+ * @returns {boolean}
+ */
+export function startInAnimationTab(state) {
+  return state.pageConstants.startInAnimationTab;
+}
```
Comment (islemaster) [inline apps/src/gamelab/stateQueries.js:57]: :+1:

**Comment by caleybrock** [comment]
I haven't noticed that in this case anymore than normal, but I'll take a look before merging.

**Comment by caleybrock** [comment]
rebuild and am seeing the issue you mentioned very consistently now :( Going to see what I can do to fix

**Comment by caleybrock** [comment]
@islemaster I added a hack that's used in applab to prevent a similar droplet rendering issue. Apparently @Bjvanminnen spent a bunch of time looking into this for applab and this was the solution that seemed to work best.

**Pull request merged**